plugins/bazaar/src/util/editBazaarProperties.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Entity } from '@backstage/catalog-model';
+import { JsonObject } from '@backstage/config';
+
+export const deleteProperties = (clonedEntity: Entity) => {
+  delete clonedEntity.metadata.uid;
+  delete clonedEntity.metadata.generation;
+  delete clonedEntity.metadata.etag;
+  delete clonedEntity.relations;
+
+  const annotationsToRemove: string[] = [
```

Comment:
Oof, this is quite brittle -- there could be many other annotations added by processors that should be removed. Really you'd want to augment the input source before catalog processing, which I think we had in mind for the newer `catalog-backend` 🤔  Not sure if that's available yet, though.